packages/fxa-admin-panel/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "FxA Admin Panel",
   "scripts": {
     "build-css": "tailwindcss -i ./src/styles/tailwind.css -o ./src/styles/tailwind.out.css --postcss",
-    "build:client": "tsc --build ../fxa-react && tsc --build && NODE_ENV=production npm run build-css && SKIP_PREFLIGHT_CHECK=true PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false rescripts build",
+    "build:client": "tsc --build ../fxa-react && tsc --build && NODE_ENV=production npm run build-css && SKIP_PREFLIGHT_CHECK=true PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false NODE_OPTIONS=--openssl-legacy-provider rescripts build",
```

Comment:
I think I'm still mildly confused by this.
Per your https://github.com/webpack/webpack/issues/14532 link the other day, that should have been fixed by https://github.com/webpack/webpack/pull/14584 (and in available since webpack v5.61.0). 
Is it better to use this legacy provider versus bumping our webpack version? Or are we stuck on webpack@4.46.0 because of CRA or some other tooling?


```sh
yarn why webpack
├─ @nestjs/cli@npm:9.1.3
│  └─ webpack@npm:5.74.0 [190f6] (via npm:>=4.0.0 <6.0.0 [190f6])
... # ignore all @storybook/* stuff
│
├─ @types/webpack@npm:5.28.0
│  └─ webpack@npm:5.64.1 [0dc39] (via npm:^5 [0dc39])
│
├─ fxa-auth-server@workspace:packages/fxa-auth-server
│  └─ webpack@npm:4.46.0 [07474] (via npm:^4.43.0 [07474])
│
├─ fxa-content-server@workspace:packages/fxa-content-server
│  └─ webpack@npm:4.46.0 [80c17] (via npm:^4.41.2 [80c17])
│
├─ fxa-pairing-channel@npm:1.0.2
│  └─ webpack@npm:4.46.0 [07474] (via npm:^4.43.0 [07474])
│
├─ fxa-payments-server@workspace:packages/fxa-payments-server
│  └─ webpack@npm:4.46.0 [07474] (via npm:^4.43.0 [07474])
│
├─ fxa-react@workspace:packages/fxa-react
│  └─ webpack@npm:4.46.0 [07474] (via npm:^4.43.0 [07474])
│
├─ fxa-settings@workspace:packages/fxa-settings
│  └─ webpack@npm:4.46.0 [07474] (via npm:^4.43.0 [07474])
│
├─ react-scripts@npm:4.0.3
│  └─ webpack@npm:4.44.2 (via npm:4.44.2)
│
└─ react-scripts@npm:4.0.3 [2acfe]
   └─ webpack@npm:4.44.2 [d1805] (via npm:4.44.2 [d1805])
```

```sh
npm info webpack@4.46 time | grep 4.46.0
#  '4.46.0': '2021-01-11T13:42:43.018Z',
```

Reply:
> Or are we stuck on webpack@4.46.0 because of CRA or some other tooling?

Yes.